xapian-core/.gitignore

```diff
@@ -37,3 +37,4 @@
 /xapian-core-*.tar.xz
 /xapian-core.spec
 /xapian-config.1
+/a.out*
```

Comment:
Where does this come from?

Reply:
The `a.out*` files are generated when running `emmake make` in the `xapian_core` folder.